### DISCUSSION TITLE
Save default Tile Size and Tile count on Settings.

### DISCUSSION THREAD

**Discussion by gurukast** [BlafKing/sd-civitai-browser-plus#50] [Ideas]
I sometimes end up refreshing the webpage (localhost) for one reason or the other, easily multiple times in a session, and [Tile Size] resets to 8, and the [Tile Count] resets to 15. I have to keep changing them, because that's a bit tiny on my monitor.  The ability to set them in settings so they always default to what i like would be nice.

**Comment by gurukast** [comment]
Also I guess the same for [Save tags after download] and [NSFW Content]?

**Comment by BlafKing** [comment]
These are the default values, WebUI has a tab to change the default value(s) to your current selected settings. 
You can change it by going into the `Settings` tab of SD-WebUI and then going to the `Defaults` tab there you can apply the current settings as the default 👍🏻 (this works for all extensions/settings of webUI)

Alternatively you could also edit the `config.json` file in your webui folder.

**Reply by gurukast** [reply]
Thanks!
